### PR TITLE
Add message about mambaforge sunsetting

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

See https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/

### CODE REVIEW

File: ubuntu/mambaforge_deprecation.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+fi
+
+case $(date +%F) in 
+    # Brownouts
+    2024-10-01|2024-10-15|2024-11-01|2024-11-10|2024-11-20|2024-11-30|2024-12-05|2024-12-10|2024-12-15|2024-12-20|2024-12-25|2024-12-30|2024-12-31|2025-*)
+        exit 1
+    ;;
+    *)
+        echo "Sleeping for 30s..."
+        sleep 30
```
Comment (hmaarrfk) [inline ubuntu/mambaforge_deprecation.sh:19]:
```suggestion
        echo "Sleeping for 300s..."
        sleep 300
```

i don't think a 30 second fluctuation in build times will be noticed. 5 mins might.... 20 mins will be!

**Review by hmaarrfk — APPROVED** [review]

File: ubuntu/mambaforge_deprecation.sh (hunk 1 above)
Comment (jaimergp) [reply ubuntu/mambaforge_deprecation.sh:19]: Fair point. We should also update this in the Miniforge repo then.

File: ubuntu/mambaforge_deprecation.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+fi
+
+case $(date +%F) in 
+    # Brownouts
+    2024-10-01|2024-10-15|2024-11-01|2024-11-10|2024-11-20|2024-11-30|2024-12-05|2024-12-10|2024-12-15|2024-12-20|2024-12-25|2024-12-30|2024-12-31|2025-*)
+        exit 1
+    ;;
+    *)
+        echo "Sleeping for 300s..."
+        for i in {1..10}; do print "."; done
```
Comment (jaimergp) [inline ubuntu/mambaforge_deprecation.sh:19]:
```suggestion
        for i in {1..300}; do sleep 1; printf "."; done
        echo
```

Did you mean that @hmaarrfk? I applied the suggestion before but I was only seeing the first edit with `sleep 300`, not this `for` loop.

**Comment by xhochy** [comment]
Do you still want this merged?

File: ubuntu/mambaforge_deprecation.sh (hunk 2 above)
Comment (hmaarrfk) [reply ubuntu/mambaforge_deprecation.sh]: yes sorry for the delay

**Comment by hmaarrfk** [comment]
yes we should still merge this.

**Review by xhochy — APPROVED** [review]

**Pull request merged**